WebHostLib/static/assets/tutorial/smz3/multiworld_en.md

```diff
@@ -0,0 +1,150 @@
+# Super Metroid Setup Guide
+
+## Required Software
+
+- One of the client programs:
+  - [SNIClient](https://github.com/ArchipelagoMW/Archipelago/releases), included with the main 
+  Archipelago install. Make sure to check the box for `SNI Client - Super Metroid Patch Setup` and 
+  `SNI Client - A Link to the Past Patch Setup`
+- Hardware or software capable of loading and playing SNES ROM files
+    - An emulator capable of connecting to SNI such as:
+        - snes9x Multitroid
+          from: [snes9x Multitroid Download](https://drive.google.com/drive/folders/1_ej-pwWtCAHYXIrvs5Hro16A1s9Hi3Jz),
+        - BizHawk from: [BizHawk Website](http://tasvideos.org/BizHawk.html)
+    - An SD2SNES, FXPak Pro ([FXPak Pro Store Page](https://krikzz.com/store/home/54-fxpak-pro.html)), or other
+      compatible hardware
+- Your legally obtained Super Metroid ROM file, probably named `Super Metroid (Japan, USA).sfc` and
+  Your Japanese Zelda3 v1.0 ROM file, probably named `Zelda no Densetsu - Kamigami no Triforce (Japan).sfc`
+
+## Installation Procedures
+
+### Windows Setup
+
+1. During the installation of Archipelago, you will have been asked to install the SNI Client. If you did not do this,
+   or you are on an older version, you may run the installer again to install the SNI Client.
+2. During setup, you will be asked to locate your base ROM files. This is your Super Metroid and Zelda3 ROM files.
+3. If you are using an emulator, you should assign your Lua capable emulator as your default program for launching ROM
+   files.
+    1. Extract your emulator's folder to your Desktop, or somewhere you will remember.
+    2. Right-click on a ROM file and select **Open with...**
+    3. Check the box next to **Always use this app to open .sfc files**
+    4. Scroll to the bottom of the list and click the grey text **Look for another App on this PC**
+    5. Browse for your emulator's `.exe` file and click **Open**. This file should be located inside the folder you
+       extracted in step one.
+
+### Macintosh Setup
+
+- We need volunteers to help fill this section! Please contact **Farrak Kilhn** on Discord if you want to help.
```

Comment:
I'm not sure it's right to leave this copy-pasta in if Farrak didn't actually write this guide. Maybe remove the section or replace with your own name if you're interested in being point of contact for mac setup.

Reply:
Oops, that one slipped. I removed the Mac section.